mongodbatlas/resource_mongodbatlas_cluster_outage_simulation.go

```diff
@@ -0,0 +1,250 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mwielbut/pointy"
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	errorClusterOutageSimulationCreate  = "error starting MongoDB Atlas Cluster Outage Simulation for Project (%s), Cluster (%s): %s"
+	errorClusterOutageSimulationRead    = "error getting MongoDB Atlas Cluster Outage Simulation for Project (%s), Cluster (%s): %s"
+	errorClusterOutageSimulationDelete  = "error ending MongoDB Atlas Cluster Outage Simulation for Project (%s), Cluster (%s): %s"
+	errorClusterOutageSimulationSetting = "error setting `%s` for MongoDB Atlas Cluster Outage Simulation: %s"
+	defaultOutageFilterType             = "REGION"
+)
+
+func resourceMongoDBAtlasClusterOutageSimulation() *schema.Resource {
```

Comment:
Don't build in anything in yet, but we should think about ways about reducing scope of this resource to first check if user is inputting a minority or majority of electable node for test outage. if minority then proceed, if majority return error message that this operation is not supported.

Reply:
ignore, we opted to fully support both Minority and Majority test outages. will need to add warning + doc updates accordingly